assets/js/components/notifications/ZeroDataStateNotifications/index.js

```diff
@@ -40,21 +40,23 @@ export default function ZeroDataStateNotifications() {
 	const isGA4DashboardView = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).isGA4DashboardView()
 	);
-	const isAnalyticsConnected = useSelect( ( select ) =>
-		select( CORE_MODULES ).isModuleConnected(
-			isGA4DashboardView ? 'analytics-4' : 'analytics'
-		)
-	);
+
+	const analyticsModuleName = isGA4DashboardView
```

Comment:
Name isn't really accurate, let's change this to "slug", otherwise this should be good to go!
```suggestion
	const analyticsModuleSlug = isGA4DashboardView
```